components/member-card-editable.js

```diff
@@ -0,0 +1,34 @@
+import IdentityLink from "./identity-link";
+import Bio from "./bio";
+import { useState } from "react";
+
+export default function MemberCardEditable({ member }) {
+  const [editState, setEditState] = useState(false);
+
+  return (
+    <li className="flex flex-col gap-6 xl:flex-row" key={member.id}>
+      {/* We cannot use next/Image here since avatars are remotely loaded
+          & we do not know the full list of sources for them */}
```

Comment:
Not sure I understand this part: “we do not know the full list of sources for them”

Reply:
Hmmmm yeah I'll add some more context to this, thanks!